File: android/app/src/main/java/io/github/pytgcalls/media/Frame.java
package io.github.pytgcalls.media;

public class Frame {
    public final long ssrc;
    public final byte[] data;
    public final FrameData frameData;

    Frame(long ssrc, byte[] data, FrameData frameData) {
        this.ssrc = ssrc;
        this.data = data;
        this.frameData = frameData;
    }
}

File: wrtc/src/models/frame.cpp
//
// Created by Lauren on 21/02/25.
//

#include <wrtc/models/frame.hpp>

namespace wrtc::models {

    Frame::Frame(
        const int64_t ssrc,
        bytes::binary data,
        const FrameData frame_data
    ): ssrc(ssrc), data(std::move(data)), frame_data(frame_data) {}

} // wrtc::models
